lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -59,13 +72,55 @@ def export_kick_off(klass)
           versions :stu3
         end
 
-        reply = export_kick_off('Patient')
+        assert_export_kick_off('Patient')
+      end
 
-        assert_response_accepted(reply)
+      test 'Server shall return "202 Accepted" or "200 OK"' do
+        metadata do
+          id '03'
+          link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#bulk-data-status-request'
+          desc %(
+          )
+          versions :stu3
+        end
 
-        @content_location = reply.response[:headers]['content-location']
+        assert_export_status
+      end
+
+      private
+
+      def export_kick_off(klass)
+        headers = { accept: 'application/fhir+json', prefer: 'respond-async' }
+
+        url = ''
+        url += "/#{klass}" if klass.present?
+        url += '/$export'
+
+        @client.get(url, @client.fhir_headers(headers))
+      end
+
+      def export_status_check(url, headers = { accept: 'application/json' })
+        wait_time = 1
+        reply = nil
+
+        loop do
```

Comment:
Could we add some kind of guard against waiting indefinitely?  It could be hardcoded to something reasonable now (skip if wait time > 10 minutes or something), but eventually it should be an input that is configured and passed to teh sequence like other parameters.  In practice we will have to put bounds on reasonable response times in our test scenario, even if there isn't an actual requirement stating things have to be done within a certain amount of time.